tasks/build/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:9.4.0
+
+# Install Wine so that we can build for the 'win32' platform.
+# https://wiki.debian.org/Wine#Installation_on_Debian_Jessie_and_newer
+RUN dpkg --add-architecture i386 \
+  && apt-get update \
+  && apt-get --yes install \
+    wine \
+    wine32 \
+    libwine
+
+# Use Gaia v0.5.0 by default.
+ARG GAIA_COMMIT=decb23bed0179244e4f42d697dcd4bf759200a2d
+
+# Download Gaia.
+RUN wget --output-document=/usr/local/bin/gaia \
+  https://tendermint-packages.interblock.io/binaries/gaia_linux_amd64/gaia_$GAIA_COMMIT
```

Comment:
So we still download Gaia by commit?

Reply:
It still downloads if I specify the binary at least and this is only the linux binary. I think this could confuse devs.

Reply:
Yes, the commit is used to identify the Gaia version.

We want only the Linux binary because it's executed in the (Linux) Docker container.

Reply:
I am not sure I understand. I was under the impression, that the docker container isn't executing Gaia but it is just adding it to the files that get bundled into Voyager for each specified platform.

Reply:
Oops, you're right of course!  I'll fix that.